### ISSUE TITLE
{N} CLI 2.5.4 produce ios bundles with android resources inside.

### ISSUE THREAD

**Issue by dtopuzov** [NativeScript/nativescript-cli#2718]
**The problem:**
{N} CLI 2.5.4 produce ios bundles with android resources inside.

**Steps to reproduce:**
```
tns create HelloWorld --ng
tns build ios
tns run ios (and do some changes in app/item/item.service.ts)
tns build ios
find platforms/ios/ -type f | grep -i android
```

**Result with {N} CLI 2.5.4:**

Android specific resources are available at `platforms/ios/HelloWorld/App/App_Resources/Android` and `platforms/ios/build`. It is a severe problem because android resources are included in final packages you are going to upload in store.

**Result with {N} CLI @next** (you can install it with `npm i -g nativescript@next`):

Android specific resources are copied only to `platforms/ios/HelloWorld/App/App_Resources/Android`.
No android specific files at `platforms/ios/build` so it looks you are safe and the final package do not contains such files.

**Comment by dtopuzov** [comment]
It looks `tns build ios --release` solves the issue.
Change it to `minor`, since we so not expect someone publish in store debug configurations.

**Comment by dtopuzov** [comment]
Fixed in `nativescript@3.1.3`
